data/src/test/java/org/oppia/android/data/backends/test/MockSubtopicTest.kt

```diff
@@ -27,19 +23,7 @@ class MockSubtopicTest {
 
   @Before
   fun setUp() {
-    val client = OkHttpClient.Builder()
-    client.addInterceptor(NetworkInterceptor())
-
-    retrofit = retrofit2.Retrofit.Builder()
-      .baseUrl(NetworkSettings.getBaseUrl())
-      .addConverterFactory(MoshiConverterFactory.create())
-      .client(client.build())
-      .build()
-
-    val behavior = NetworkBehavior.create()
-    mockRetrofit = MockRetrofit.Builder(retrofit)
-      .networkBehavior(behavior)
-      .build()
+    mockRetrofit = MockRetrofitHelper().createMockRetrofit()
```

Comment:
@BenHenning Initially, I thought of passing retrofit as an argument to the method `createMockRetrofit()` but as it isn't used anywhere else, I chose to move the whole thing.